parts/kubernetesmastervars.t

```diff
@@ -64,7 +68,11 @@
     "osImageSKU": "16.04-LTS",
     "osImageVersion": "16.04.201706191",
     "resourceGroup": "[resourceGroup().name]",
+{{if .HasMaster}}
     "routeTableName": "[concat(variables('masterVMNamePrefix'),'routetable')]",
+{{else}}
+    "routeTableName": "FIXME",
```

Comment:
Could this route table live in the same resource group as the agents like it does today for a master+Agents cluster and so same naming scheme can be used?